index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
The issue here is that the console will always log the error msg, simply because you are calling the function when constructing the request object. Unfortunately there's no easy way to generate a warning from using a specific property without refactoring major parts of the project or adding niche code to support this particular feature.